<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CheckI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton, QComboBox {
	background-color: #03F003;
	color: black;
	
	
}

menubar, toolbar {
	height: 1px;
}

QWidget {
	color: #white
	background: #011001;
	font: bold 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="select_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Breasts Cancer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pneumonia</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze for</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="path_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>File: </string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="style.css"/>
 </resources>
 <connections/>
</ui>
